packages/expo-updates/ios/EXUpdates/EnabledAppController.swift

```diff
@@ -247,4 +247,11 @@ public class EnabledAppController: InternalAppControllerInterface, StartupProced
   public func getEmbeddedUpdate() -> Update? {
     return EmbeddedAppLoader.embeddedManifest(withConfig: self.config, database: self.database)
   }
+
+  public func setUpdatesURLAndRequestHeadersOverrides(_ overrides: UpdatesRuntimeOverrides?) throws {
```

Comment:
let's call this interface method


```suggestion
  public func setUpdatesURLAndRequestHeadersOverride(_ override: UpdatesConfigurationOverride?) throws {
```